jwql/website/apps/jwql/archive_database_update.py

```diff
@@ -387,6 +402,13 @@ def fill_empty_model(model_name, model_field):
     except ValueError:
         pass
 
+    # filter(field__exact=DEFAULT_MODEL_CHARFIELD)
```

Comment:
Did you want to remove this line?

Reply:
Keeping that there, that's a comment showing what the kwargs expansion does, there is one like that above each try statement since the code isn't the most legible